litellm/tests/test_notdiamond_completion.py

```diff
@@ -0,0 +1,162 @@
+
+
+def test_completion_notdiamond():
+    try:
+        litellm.set_verbose = False
+        messages = [
+            {
+                "role": "user",
+                "content": "Hey",
+            },
+        ]
+        for model in nd_model_list:
+            print("model:", f"{model['provider']}/{model['model']}")
+            response = completion(
+                model="notdiamond/notdiamond",
+                messages=messages,
+                llm_providers=[model],
+            )
+            print(response)
+    except Exception as e:
+        pytest.fail(f"Error occurred: {e}")
+
+
+def test_completion_notdiamond_stream():
+    try:
+        litellm.set_verbose = False
+        messages = [
+            {
+                "role": "user",
+                "content": "Hey",
+            },
+        ]
+        for model in nd_model_list:
+            print("model:", f"{model['provider']}/{model['model']}")
+            response = completion(
+                model="notdiamond/notdiamond",
+                messages=messages,
+                llm_providers=[model],
+                stream=True
+            )
+            print(response)
+    except Exception as e:
+        pytest.fail(f"Error occurred: {e}")
+
+
+def test_completion_notdiamond_tool_calling():
+    try:
+        litellm.set_verbose = False
+        messages = [
+            {
+                "role": "user",
+                "content": "what is 2 + 5?",
+            },
+        ]
+        tools = [
+            {
+                "type": "function",
+                "function": {
+                "name": "add",
+                "description": "Adds a and b.",
+                "parameters": {
+                    "type": "object",
+                    "properties": {"a": {"type": "integer"}, "b": {"type": "integer"}},
+                    "required": ["a", "b"],
+                },
+                },
+            },
+        ]
+        for model in nd_tools_model_list:
+            print("model:", f"{model['provider']}/{model['model']}")
+            response = completion(
+                model="notdiamond/notdiamond",
+                messages=messages,
+                llm_providers=[model],
+                tools=tools
+            )
+            print(response)
+    except Exception as e:
+        pytest.fail(f"Error occurred: {e}")
+
+
+# @pytest.mark.asyncioc
```

Comment:
Should this remain here? The method contains async logic, but I don't remember whether the `asyncio.run` necessitates annotating this test (or identifying the method as `async`).

Reply:
I don't think it does. I just found it in a few other async tests, but they seem to be inconsistent about it, so I can remove it from here.